README.md

```diff
@@ -3,10 +3,10 @@
 > The tiny framework for building hypertext applications.
 
 - **Do more with less**—We have minimized the concepts you need to learn to get stuff done. Views, actions, effects, and subscriptions are all pretty easy to get to grips with and work together seamlessly.
-- **Write what, not how**—With a declarative API that's easy to read and fun to write, Hyperapp is the best way to build purely functional, feature-rich, browser-based apps using idiomatic JavaScript.
+- **Write what, not how**—With a declarative API that’s easy to read and fun to write, Hyperapp is the best way to build purely functional, feature-rich, browser-based apps using idiomatic JavaScript.
```

Comment:
Whoopsie: `’`. Can we go back to my good old `'`?

Reply:
If you wish... 🧞‍♂️